contracts/marketplace.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please move all your NFT-related changes into a new folder like `nft` or `nft-minting` (including all of the `package.json` and `package-lock.json` file changes)

The project's folder structure should be like:
```
/crazy-bee-project
  - /docs
  - /frontend
  - /backend
  - /ai-creator
  - /nft-minting

```